x11-packages/firefox/0028-fix-headless-mode.patch

```diff
@@ -0,0 +1,14 @@
+Fixes a crash in the command "firefox -headless"
+--- a/toolkit/moz.configure
++++ b/toolkit/moz.configure
+@@ -2714,8 +2714,8 @@ def has_remote(toolkit):
+         return True
```

Comment:
It may be better to patch it here.

```patch
 @depends(toolkit)
 def has_remote(toolkit):
+    return False
     if toolkit in ("gtk", "windows", "cocoa"):
         return True
```

Reply:
@licy183 thank you for the interesting observation, 

I can let you know that unfortunately, I actually already tested a very similar patch to that first, before iterating to reach the patch as it is now, and for me it did not work. Only the way I have written it as shown in this PR worked for me.

If you would like, you can test your idea as well and let me know if you are able to get it to work that way and still fix the crash. I would be interested to know if I was just doing it wrong, and it can actually work that way.